NuRadioMC/SignalProp/radioproparaytracing.py

```diff
@@ -381,6 +392,111 @@ def delta(ray_dir,shower_dir):
     def set_minimizer_tolerance(self, xtol=1e-3*units.deg, ztol=1e-3*units.meter):
         self.__xtol = xtol
         self.__ztol = ztol
+
+    def raytracer_birefringence(self, launch_v, spec, s_rate):
+        """
+        Uses RadioPropa propagate an electric field through the correct solution of the ray path. To use add to the config.yaml file:
+        propagation:
```

Comment:
indentation seems wrong

Reply:
Here I'm not sure what you mean. The indentation of the function itself or of the documentation? Both look normal to me.